qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageLocationAttributeTypesPage.java

```diff
@@ -105,12 +109,17 @@ public void cancelPurgeLocation() {
 	}
 
 	public void retireLocation() {
-		clickOn(RETIRE_LOCATION_ICON);
+		clickOnLast(RETIRE_LOCATION_ICON);
```

Comment:
Here also it will be better if we can delete a specific `location type attribute` instead of removing the last one on the table. Then we can check for the name of the deleted location type attribute` and ensure that it is deleted successfully.

Reply:
refer to https://github.com/openmrs/openmrs-contrib-qaframework/pull/433#discussion_r940895960